workers/generic-worker/helper_test.go

```diff
@@ -372,6 +372,7 @@ func GWTest(t *testing.T) *Test {
 			// These ports are not exposed outside of the host. However, in CI they must differ from those of the
 			// generic-worker instance running the test suite.
 			LiveLogPortBase:    30583,
+			MaximumMaxRunTime:  86400,
```

Comment:
This is a bit weird. When I run the test I get a 0 for this value (that's why I set it again here) but when I do `generic-worker run` locally I see it defaulting to 86400